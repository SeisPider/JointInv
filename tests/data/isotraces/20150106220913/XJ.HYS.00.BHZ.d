SURF96 R U X -1 94.99999 3.68704 0.24409
SURF96 R U X -1 75.00000 3.66342 0.19024
SURF96 R U X -1 75.00000 3.88734 0.21421
SURF96 R U X -1 70.00000 3.60502 0.17194
SURF96 R U X -1 70.00000 3.84496 0.19559
SURF96 R U X -1 65.00000 3.73884 0.17173
SURF96 R U X -1 60.00000 3.62900 0.14935
SURF96 R U X -1 55.00000 3.62332 0.13647
SURF96 R U X -1 50.00000 3.57695 0.12091
SURF96 R U X -1 48.00000 3.56650 0.11540
SURF96 R U X -1 46.00000 3.55914 0.11013
SURF96 R U X -1 44.00000 3.54435 0.10447
SURF96 R U X -1 42.00000 3.51425 0.09804
SURF96 R U X -1 40.00000 3.46915 0.09099
SURF96 R U X -1 38.00000 3.41446 0.08373
SURF96 R U X -1 36.00000 3.34933 0.07633
SURF96 R U X -1 34.00000 3.28376 0.06929
SURF96 R U X -1 32.00000 3.23602 0.06333
SURF96 R U X -1 30.00000 3.18824 0.05764
SURF96 R U X -1 29.00000 3.16442 0.05489
SURF96 R U X -1 29.00000 4.02150 0.08864
SURF96 R U X -1 27.00000 3.12343 0.04978
SURF96 R U X -1 26.00000 3.10102 0.04726
SURF96 R U X -1 25.00000 3.06526 0.04440
