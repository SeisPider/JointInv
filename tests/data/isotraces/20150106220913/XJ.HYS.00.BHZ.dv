MFT96 R U T -1 95.256    3.7113    1.0000    5290.9     0.6063E-06   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 93.091    3.7146    1.0000    5290.9     0.6254E-06   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 91.022    3.7175    1.0000    5290.9     0.6380E-06   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 89.043    3.7200    1.0000    5290.9     0.6456E-06   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 87.149    3.7221    1.0000    5290.9     0.6505E-06   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 85.333    3.7239    1.0000    5290.9     0.6555E-06   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 83.592    3.7253    1.0000    5290.9     0.6640E-06   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 81.920    3.7264    1.0000    5290.9     0.6799E-06   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 80.314    3.7271    1.0000    5290.9     0.7068E-06   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 78.769    3.7275    1.0000    5290.9     0.7483E-06   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 77.283    3.7275    1.0000    5290.9     0.8068E-06   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 75.852    3.7272    1.0000    5290.9     0.8841E-06   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 74.473    3.7266    1.0000    5290.9     0.9812E-06   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 73.143    3.7257    1.0000    5290.9     0.1099E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 71.860    3.7244    1.0000    5290.9     0.1236E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 70.621    3.7229    1.0000    5290.9     0.1394E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 69.424    3.7210    1.0000    5290.9     0.1573E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 68.267    3.7188    1.0000    5290.9     0.1771E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 67.148    3.7164    1.0000    5290.9     0.1988E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 66.065    3.7137    1.0000    5290.9     0.2224E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 65.016    3.7106    1.0000    5290.9     0.2478E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 64.000    3.7074    1.0000    5290.9     0.2750E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 63.015    3.7038    1.0000    5290.9     0.3037E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 62.061    3.7000    1.0000    5290.9     0.3338E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 61.134    3.6960    1.0000    5290.9     0.3654E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 60.235    3.6917    1.0000    5290.9     0.3981E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 59.362    3.6872    1.0000    5290.9     0.4320E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 58.514    3.6825    1.0000    5290.9     0.4667E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 57.690    3.6775    1.0000    5290.9     0.5022E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 56.889    3.6724    1.0000    5290.9     0.5383E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 56.110    3.6670    1.0000    5290.9     0.5748E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 55.351    3.6614    1.0000    5290.9     0.6115E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 54.613    3.6557    1.0000    5290.9     0.6481E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 53.895    3.6497    1.0000    5290.9     0.6845E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 53.195    3.6436    1.0000    5290.9     0.7204E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 52.513    3.6373    1.0000    5290.9     0.7555E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 51.848    3.6309    1.0000    5290.9     0.7898E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 51.200    3.6243    1.0000    5290.9     0.8230E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 50.568    3.6175    1.0000    5290.9     0.8549E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 49.951    3.6106    1.0000    5290.9     0.8856E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 49.349    3.6036    1.0000    5290.9     0.9150E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 48.762    3.5965    1.0000    5290.9     0.9431E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 48.188    3.5892    1.0000    5290.9     0.9702E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 47.628    3.5818    1.0000    5290.9     0.9965E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 47.080    3.5743    1.0000    5290.9     0.1022E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 46.545    3.5668    1.0000    5290.9     0.1048E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 46.022    3.5591    1.0000    5290.9     0.1074E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 45.511    3.5513    1.0000    5290.9     0.1100E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 45.011    3.5435    1.0000    5290.9     0.1128E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 44.522    3.5356    1.0000    5290.9     0.1157E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 44.043    3.5276    1.0000    5290.9     0.1188E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 43.574    3.5196    1.0000    5290.9     0.1221E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 43.116    3.5115    1.0000    5290.9     0.1257E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 42.667    3.5033    1.0000    5290.9     0.1295E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 42.227    3.4951    1.0000    5290.9     0.1336E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 41.796    3.4869    1.0000    5290.9     0.1380E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 41.374    3.4786    1.0000    5290.9     0.1426E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 40.960    3.4704    1.0000    5290.9     0.1473E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 40.554    3.4621    1.0000    5290.9     0.1522E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 40.157    3.4537    1.0000    5290.9     0.1571E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 39.767    3.4454    1.0000    5290.9     0.1621E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 39.385    3.4371    1.0000    5290.9     0.1670E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 39.010    3.4287    1.0000    5290.9     0.1719E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 38.642    3.4204    1.0000    5290.9     0.1766E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 38.280    3.4121    1.0000    5290.9     0.1812E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 37.926    3.4038    1.0000    5290.9     0.1856E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 37.578    3.3955    1.0000    5290.9     0.1899E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 37.236    3.3872    1.0000    5290.9     0.1939E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 36.901    3.3790    1.0000    5290.9     0.1979E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 36.571    3.3708    1.0000    5290.9     0.2018E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 36.248    3.3626    1.0000    5290.9     0.2057E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 35.930    3.3545    1.0000    5290.9     0.2096E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 35.617    3.3465    1.0000    5290.9     0.2138E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 35.310    3.3384    1.0000    5290.9     0.2181E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 35.009    3.3305    1.0000    5290.9     0.2229E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 34.712    3.3225    1.0000    5290.9     0.2280E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 34.420    3.3147    1.0000    5290.9     0.2336E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 34.133    3.3069    1.0000    5290.9     0.2397E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 33.851    3.2992    1.0000    5290.9     0.2463E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 33.574    3.2916    1.0000    5290.9     0.2535E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 33.301    3.2840    1.0000    5290.9     0.2611E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 33.032    3.2765    1.0000    5290.9     0.2692E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 32.768    3.2691    1.0000    5290.9     0.2775E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 32.508    3.2618    1.0000    5290.9     0.2861E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 32.252    3.2546    1.0000    5290.9     0.2947E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 32.000    3.2474    1.0000    5290.9     0.3033E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 31.752    3.2404    1.0000    5290.9     0.3118E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 31.508    3.2335    1.0000    5290.9     0.3200E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 31.267    3.2267    1.0000    5290.9     0.3279E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 31.030    3.2199    1.0000    5290.9     0.3355E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 30.797    3.2133    1.0000    5290.9     0.3426E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 30.567    3.2068    1.0000    5290.9     0.3495E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 30.341    3.2004    1.0000    5290.9     0.3560E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 30.118    3.1941    1.0000    5290.9     0.3623E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 29.898    3.1880    1.0000    5290.9     0.3684E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 29.681    3.1820    1.0000    5290.9     0.3744E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 29.468    3.1761    1.0000    5290.9     0.3805E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 29.257    3.1703    1.0000    5290.9     0.3866E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 29.050    3.1647    1.0000    5290.9     0.3930E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 28.845    3.1592    1.0000    5290.9     0.3996E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 28.643    3.1538    1.0000    5290.9     0.4065E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 28.444    3.1486    1.0000    5290.9     0.4136E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 28.248    3.1435    1.0000    5290.9     0.4208E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 28.055    3.1385    1.0000    5290.9     0.4280E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 27.864    3.1337    1.0000    5290.9     0.4351E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 27.676    3.1291    1.0000    5290.9     0.4419E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 27.490    3.1246    1.0000    5290.9     0.4479E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 27.307    3.1203    1.0000    5290.9     0.4531E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 27.126    3.1161    1.0000    5290.9     0.4570E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 26.947    3.1121    1.0000    5290.9     0.4593E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 26.771    3.1082    1.0000    5290.9     0.4598E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 26.597    3.1046    1.0000    5290.9     0.4580E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 26.426    3.1011    1.0000    5290.9     0.4539E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 26.256    3.0977    1.0000    5290.9     0.4472E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 26.089    3.0945    1.0000    5290.9     0.4377E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 25.924    3.0916    1.0000    5290.9     0.4253E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 25.761    3.0887    1.0000    5290.9     0.4102E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 25.600    3.0861    1.0000    5290.9     0.3923E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 25.441    3.0837    1.0000    5290.9     0.3720E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 25.284    3.0814    1.0000    5290.9     0.3493E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 25.129    3.0794    1.0000    5290.9     0.3248E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 95.000    3.7117    1.0000    5290.9     0.6086E-06   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 90.000    3.7188    1.0000    5290.9     0.6420E-06   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 85.000    3.7242    1.0000    5290.9     0.6571E-06   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 80.000    3.7272    1.0000    5290.9     0.7152E-06   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 75.000    3.7268    1.0000    5290.9     0.9440E-06   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 70.000    3.7219    1.0000    5290.9     0.1487E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 65.000    3.7106    1.0000    5290.9     0.2483E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 60.000    3.6905    1.0000    5290.9     0.4072E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 55.000    3.6587    1.0000    5290.9     0.6289E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 50.000    3.6112    1.0000    5290.9     0.8832E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 48.000    3.5867    1.0000    5290.9     0.9791E-05   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 46.000    3.5588    1.0000    5290.9     0.1075E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 44.000    3.5269    1.0000    5290.9     0.1191E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 42.000    3.4908    1.0000    5290.9     0.1359E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 40.000    3.4504    1.0000    5290.9     0.1591E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 38.000    3.4055    1.0000    5290.9     0.1847E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 36.000    3.3563    1.0000    5290.9     0.2088E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 34.000    3.3033    1.0000    5290.9     0.2428E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 32.000    3.2474    1.0000    5290.9     0.3033E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 30.000    3.1909    1.0000    5290.9     0.3655E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 29.000    3.1633    1.0000    5290.9     0.3946E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 28.000    3.1372    1.0000    5290.9     0.4301E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 27.000    3.1133    1.0000    5290.9     0.4586E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 26.000    3.0929    1.0000    5290.9     0.4310E-04   55.204899  163.244598   42.976002   89.441002 0  1 COMMENT
