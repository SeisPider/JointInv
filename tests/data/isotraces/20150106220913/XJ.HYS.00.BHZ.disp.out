SURF96 R U T -1 95.000    3.7117    1.0000    
SURF96 R C T -1 95.000    6.4982    1.0000    
SURF96 R U T -1 90.000    3.7188    1.0000    
SURF96 R C T -1 90.000    6.2520    1.0000    
SURF96 R U T -1 85.000    3.7242    1.0000    
SURF96 R C T -1 85.000    6.0243    1.0000    
SURF96 R U T -1 80.000    3.7272    1.0000    
SURF96 R C T -1 80.000    5.8133    1.0000    
SURF96 R U T -1 75.000    3.7268    1.0000    
SURF96 R C T -1 75.000    5.6169    1.0000    
SURF96 R U T -1 70.000    3.7219    1.0000    
SURF96 R C T -1 70.000    5.4329    1.0000    
SURF96 R U T -1 65.000    3.7106    1.0000    
SURF96 R C T -1 65.000    5.2593    1.0000    
SURF96 R U T -1 60.000    3.6905    1.0000    
SURF96 R C T -1 60.000    5.0944    1.0000    
SURF96 R U T -1 55.000    3.6587    1.0000    
SURF96 R C T -1 55.000    4.9356    1.0000    
SURF96 R U T -1 50.000    3.6112    1.0000    
SURF96 R C T -1 50.000    4.7802    1.0000    
SURF96 R U T -1 48.000    3.5867    1.0000    
SURF96 R C T -1 48.000    4.7182    1.0000    
SURF96 R U T -1 46.000    3.5588    1.0000    
SURF96 R C T -1 46.000    4.6560    1.0000    
SURF96 R U T -1 44.000    3.5269    1.0000    
SURF96 R C T -1 44.000    4.5933    1.0000    
SURF96 R U T -1 42.000    3.4908    1.0000    
SURF96 R C T -1 42.000    4.5297    1.0000    
SURF96 R U T -1 40.000    3.4504    1.0000    
SURF96 R C T -1 40.000    4.4648    1.0000    
SURF96 R U T -1 38.000    3.4055    1.0000    
SURF96 R C T -1 38.000    4.3983    1.0000    
SURF96 R U T -1 36.000    3.3563    1.0000    
SURF96 R C T -1 36.000    4.3297    1.0000    
SURF96 R U T -1 34.000    3.3033    1.0000    
SURF96 R C T -1 34.000    4.2586    1.0000    
SURF96 R U T -1 32.000    3.2474    1.0000    
SURF96 R C T -1 32.000    4.1847    1.0000    
SURF96 R U T -1 30.000    3.1909    1.0000    
SURF96 R C T -1 30.000    4.1076    1.0000    
SURF96 R U T -1 29.000    3.1633    1.0000    
SURF96 R C T -1 29.000    4.0679    1.0000    
SURF96 R U T -1 28.000    3.1372    1.0000    
SURF96 R C T -1 28.000    4.0274    1.0000    
SURF96 R U T -1 27.000    3.1133    1.0000    
SURF96 R C T -1 27.000    3.9863    1.0000    
SURF96 R U T -1 26.000    3.0929    1.0000    
SURF96 R C T -1 26.000    3.9447    1.0000    
